indexer-app/src/cmr/indexer/data/concepts/variable.clj

```diff
@@ -81,16 +90,27 @@
        :metadata-format (name (mt/format-key format))
        :measurement-identifiers (mapcat measurement-identifier->elastic-doc
                                         (:MeasurementIdentifiers parsed-concept))
-       ;; associated collections saved in elasticsearch for retrieving purpose in the format of:
+       ;; associated collections and generic concepts saved in elasticsearch for retrieving purpose in the format of:
        ;; [{"concept_id":"C1200000007-PROV1"}, {"concept_id":"C1200000008-PROV1","revision_id":5}]
-       :collections-gzip-b64 (when (seq variable-associations)
-                               (util/string->gzip-base64
-                                (pr-str
-                                 (map (fn [va]
-                                        (util/remove-nil-keys
-                                         {"concept-id" (:associated-concept-id va)
-                                          "revision-id" (:associated-revision-id va)}))
-                                      variable-associations))))})))
+       :concepts-gzip-b64 (when (or (seq variable-associations)
+                                    (seq generic-associations))
+                            (util/string->gzip-base64
+                             (pr-str
+                              (util/remove-map-keys empty?
+                               {:collections (map #(get-concept-id-revision-id-map
+                                                    (:associated-concept-id %)
+                                                    (:associated-revision-id %))
+                                                  variable-associations)
+                                :generics (let [generics (concat
```

Comment:
I think this section of the index has been changed by Erich. I think this might be the section that he now breaks out by type, but I'll have to go check.